keras/layers/wrappers.py

```diff
@@ -487,12 +487,33 @@ def updates(self):
             return self.forward_layer.updates + self.backward_layer.updates
         return []
 
+    def get_updates_for(self, inputs=None):
```

Comment:
Why not

```python
forward_updates = self.forward_layer.get_updates_for(inputs)
backward_updates = self.backward_layer.get_updates_for(inputs)
return super(Wrapper, self).get_updates_for(inputs) + forward_updates + backward_updates
```

Indeed, the inner layers are called with the parent's `inputs` argument directly.